script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
Consider using 'with open' in a more efficient way by directly loading the JSON file without reading it entirely into memory first.
```suggestion
with open('scripts/example_data.json') as json_file:
    data = json.load(json_file)
    print(json.dumps(data, indent=2))
```